change: Rename node and runtime crates (#621)
input-output-hk/partner-chains: demo/README.md

# Partner Chains demo implementation

> This directory contains a reference implementation of a complete Partner Partner Chain.

Reviewer: ```suggestion
This directory contains a reference implementation of a complete Partner Chains node.
```

It consists of a runtime crate showcasing use of all Partner Chains SDK pallets and
a node implementation demonstrating how all node components, data sources and RPCs provided

> by the Partner Chains SDK should be composed into a Substrate node.

Reviewer 2: ?
```suggestion
by the Partner Chains SDK could be composed into a Substrate node.
```

This reference implementation is provided on a **best-effort basis**. Partner Chain builders

> should always verify that examples contained in this directory is relevant to their

Reviewer: ```suggestion
should always verify that examples contained in this directory are relevant to their
```
